### PR TITLE
[AR-139] For all Textual Work Records in Sales and Knoedler make sure that each record has both a form and a content type

### PR DESCRIPTION
**Pull request by mafragias**

### CODE REVIEW

File: pipeline/projects/sales/catalogs.py [hunk 1]
```diff
@@ -217,7 +218,7 @@ def __call__(self, data:dict, non_auctions):
 
 		catalog_label = self.helper.physical_catalog_label(cno, sale_type, owner, copy)
 		row_name = f'STAR Entry for Physical {catalog_label}'
-		row = vocab.EntryTextForm(ident=record_uri, content=content, label=row_name)
+		row = vocab.make_multitype_obj(vocab.EntryTextForm,self.helper.catalog_type(sale_type), ident=record_uri, content=content, label=row_name)
```
Comment (kasei) [inline pipeline/projects/sales/catalogs.py:221]: I'm not sure this one should have typing relating to the sale type. This is an entry in the Getty database of known physical copies of catalogs (an entry in the STAR database). All such entries seem like they should have the same typing, regardless of the type of event the catalog they describe is representing.

File: pipeline/projects/sales/objects.py [hunk 2]
```diff
@@ -139,8 +139,12 @@ def _populate_object_catalog_record(self, data:dict, parent, lot, cno, rec_num,
 		creation.carried_out_by = self.helper.static_instances.get_instance('Group', 'gpi')
 		row.created_by = creation
 		row.identified_by = self.helper.gpi_number_id(rec_num, vocab.StarNumber)
-
-		record = vocab.EntryTextForm(ident=record_uri, label=f'Sale recorded in catalog: {lot_object_id} (record number {rec_num})')
+		
+		sale_type = non_auctions.get(cno, data.get('non_auction_flag'))
```
Comment (kasei) [inline pipeline/projects/sales/objects.py:143]: The `non_auction_flag` only exists in the sales descriptions (events) data, so this `data.get` call will always return `None`.

File: pipeline/projects/sales/objects.py [hunk 3]
```diff
@@ -139,8 +139,12 @@ def _populate_object_catalog_record(self, data:dict, parent, lot, cno, rec_num,
...
+		if sale_type:
+			non_auctions[cno] = sale_type
```
Comment (kasei) [inline pipeline/projects/sales/objects.py:145]: What is the point of setting `non_auctions[cno]` here?

**Review by kasei — CHANGES_REQUESTED** [review]

File: pipeline/projects/sales/objects.py (hunk 2 above)
Comment (mafragias) [reply pipeline/projects/sales/objects.py]: After testing that with an other type, "Private Contract Sale", I got the proper type for that record, and did not get None

**Pull request closed without merging**